pages/register/register.view.html

```diff
@@ -0,0 +1,47 @@
+<div ng-controller="RegisterController as Register" class="register-page">
+	
+	<div ng-if="Register.loading">{{'LOADING' | translate}}</div>
+	
+	<article>
+		<div class="register-logo">
+			<img src="../../img/brand/vimojo.svg" alt="vimojo_logo" />
+			<h2 ng-if="!Register.error"><pre>{{'REGISTER_SLOGAN' | translate}}</pre></h2>	
+		</div>
+
+		<section class="register-form">
+			<form name="form" novalidate>
+				<div ng-if="Register.status.register">
+					<input type="text" placeholder="{{'USERNAME' | translate }}" ng-model="Register.username">
+					<input type="text" placeholder="{{'EMAIL' | translate }}" ng-model="Register.email">
+					<input type="password" placeholder="{{'PASSWORD' | translate }}" ng-model="Register.password">
+				</div>
+				<div ng-if="Register.status.terms">
+					<input type="text" placeholder="{{'QUESTION_AGE' | translate }}" ng-model="Register.age">
+					<span>
+						<input type="checkbox" id="signup" ng-model="Register.terms">
+						<label for="signup">{{'ACCEPT_CONDITIONS_1' | translate}}<span>
+							<a ui-sref="terms" target="_blank">{{'CONDITIONS' | translate}}</a>
+						</span>{{'ACCEPT_CONDITIONS_2' | translate}}</label>
+					</span>
+				</div>
+				<div ng-if="Register.status.captcha" class="g-recaptcha" data-sitekey="6LeELkIUAAAAACXrqJCDeOlCvvio78mgG2wj22ou"></div>
+			</form>
+			<ul class="credentials-errors" ng-if="Register.error" ng-bind-html="Register.error | renderHTMLCorrectly"></ul>
```

Comment:
As I've previously said, **be careful** with this kind of things. Do we need HTML for the error message? Is there a way in what the user might write it's own text here? I've not tried to attack, in this concrete case, so I'm not sure what are the right answers. Just to remind the importance of thinking twice this kind of actions 🦊

Reply:
at the beginning, I was throwing the string scaped but later I realized some errors could be longer than the max size of the container, so `word-wrap` & `line-height` CSS properties allowed me to display errors more beautiful

Reply:
And what about an array? 4c086b3